R/plumb-block.R

```diff
@@ -151,6 +152,30 @@ plumbBlock <- function(lineNum, file){
 
     }
 
+    parsersMat <- stri_match(line, regex="^#['\\*]\\s*@parser(\\s+([^\\s]+)\\s*(.*)\\s*$)?")
+    if (!is.na(parsersMat[1,1])){
+      parser_alias <- stri_trim_both(parsersMat[1,3])
+      if (is.na(parser_alias) || parser_alias == ""){
+        stopOnLine(lineNum, line, "No @parser specified")
+      }
+
+      if (!parser_alias %in% registered_parsers()){
+        stopOnLine(lineNum, line, paste0("No such @parser registered: ", parser_alias))
+      }
+
+      if (!is.na(parsersMat[1, 4]) && parsersMat[1,4] != ""){
+        # We have an arg to pass in to the parser
+        arg_list <- eval(parse(text=parsersMat[1,4]))
```

Comment:
add envir to 
```r
arg_list <- eval(parse(text=parsersMat[1,4]))
...
arg_list <- eval(parse(text=parsersMat[1,4]), envir)
```

#620

Reply:
Should also prolly surround each `eval` statement in a `tryCatch` with a `stopOnLine` in the error part.